otri/database/database_adapter.py

```diff
@@ -192,3 +177,23 @@ def _connection_string(self) -> str:
             with the user info.
         '''
         raise NotImplementedError("This is an abstract method, please implement it in a sub-class")
+
+    def _make_engine(self, conn_str: str) -> Engine:
```

Comment:
Codacy found an issue: [Method could be a function](https://app.codacy.com/gh/OTRI-Unipd/OTRI/pullRequest?prid=5998990)